src/containers/flows/AcceptBid/AcceptBid.tsx

```diff
@@ -0,0 +1,278 @@
+//- React Imports
+import { useEffect, useRef, useState } from 'react';
+
+//- Global Component Imports
+import { Overlay, Wizard } from 'components';
+
+//- Components Imports
+import Details from './components/Details/Details';
+import Success from './components/Success/Success';
+
+//- Library Imports
+import useAcceptBid from './hooks/useAcceptBid';
+import { useWeb3React } from '@web3-react/core';
+import { useZnsSdk } from 'lib/providers/ZnsSdkProvider';
+import { Bid } from '@zero-tech/zauction-sdk';
+import { Metadata } from 'lib/types';
+
+//- Types Imports
+import { Step } from './AcceptBid.types';
+
+//- Constants Imports
+import {
+	BUTTONS,
+	ERRORS,
+	MESSAGES,
+	STATUS_TEXT,
+	TITLES,
+} from './AcceptBid.constants';
+
+type AcceptBidProps = {
+	acceptingBid: Bid | undefined;
+	domainMetadata: Metadata | undefined;
+	refetch?: (bid: Bid) => void;
+	isLoading?: boolean;
+	assetUrl: string;
+	creatorId: string;
+	domainTitle: string;
+	domainId: string;
+	domainName: string;
+	walletAddress: string;
+	highestBid?: string;
+	wildPriceUsd?: number;
+	onClose: () => void;
+};
+
+const AcceptBid = ({
+	acceptingBid,
+	refetch,
+	isLoading,
+	assetUrl,
+	creatorId,
+	domainTitle,
+	domainId,
+	domainName,
+	walletAddress,
+	highestBid,
+	wildPriceUsd,
+	onClose,
+}: AcceptBidProps) => {
+	//////////////////
+	// State & Data //
+	//////////////////
+
+	// Hooks
+	const { accept, status } = useAcceptBid();
+	const { instance: sdk } = useZnsSdk();
+	const { account, library } = useWeb3React();
+
+	const [currentStep, setCurrentStep] = useState<Step>(Step.LoadingData);
+	const [error, setError] = useState<string | undefined>();
+	const [stepTitle, setStepTitle] = useState<string>(
+		TITLES[Step.Details].PRIMARY,
+	);
```

Comment:
nit: I think you could just use `TITLES[currentStep]` (after making all of the values `string` rather than `object`) directly in the `<Wizard>` `header` prop - this would save needing to `setStepTitle` on step changes.

Reply:
tidied this up on accept bid & transfer ownership flow